New Checkout Drop-in v1.4.0-beta1 (#254)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/index.mdx

Visit the [Title and header customization](/dropins/checkout/containers/overview/#title-and-header-customization) documentation for more information.

The `LoginForm` container configuration structure has been modified. New properties are added to the `LoginFormProps` interface:

- **`LoginFormProps.displayHeadingContent` property added**: This property allows displaying the container heading content.
- **`LoginFormProps.slots.Heading` property added**: This property allows customizing the container heading content based on the user authentication status.

And `LoginFormProps` interface is extending from `TitleProps` interface inheriting the following properties:

- **`LoginFormProps.displayTitle` property inherited**: This property allows displaying the container title.
- **`LoginFormProps.slots.Title` property inherited**: This property allows customizing the container title.

Visit the [LoginForm](/dropins/checkout/containers/login-form) documentation for more information.

The `PaymentMethods` container configuration structure has been modified. New properties are added to the `PaymentMethodsProps` interface:

> - **`PaymentMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as selector for each payment method; UI components available are: 'ToggleButton' and 'RadioButton'.

Reviewer: ```suggestion
- **`PaymentMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as a selector for each payment method. UI components available are: 'ToggleButton' and 'RadioButton'.
```

> - **`PaymentMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as selector for each payment method; UI components available are: 'ToggleButton' and 'RadioButton'.

Reviewer: ```suggestion
- **`PaymentMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as a selector for each payment method. UI components available are: 'ToggleButton' and 'RadioButton'.
```

And `PaymentMethodsProps` interface is extending from `TitleProps` interface inheriting the following properties:

- **`PaymentMethodsProps.displayTitle` property inherited**: This property allows displaying the container title.
- **`PaymentMethodsProps.slots.Title` property inherited**: This property allows customizing the container title.

Visit the [PaymentMethods](/dropins/checkout/containers/payment-methods) documentation for more information.

> The `ShippingMethods` container configuration structure has been modified. `ShippingMethodsProps` interface is extending from `TitleProps` interface inheriting the following properties:

Reviewer: ```suggestion
The `ShippingMethods` container configuration structure has been modified. The `ShippingMethodsProps` interface extends the `TitleProps` interface inheriting the following properties:
```

> - **`ShippingMethodsProps.displayTitle` property inherited**: This property allows displaying the container title.

Reviewer: ```suggestion
- **`ShippingMethodsProps.displayTitle` property inherited**: This property displays or hides the container title.
```

- **`ShippingMethodsProps.slots.Title` property inherited**: This property allows customizing the container title.

Visit the [ShippingMethods](/dropins/checkout/containers/shipping-methods) documentation for more information.

> The `TermsAndConditions` container now is visible for both anonymous and authenticated checkout process.

Reviewer: ```suggestion
The `TermsAndConditions` container is now visible for both the anonymous and authenticated checkout processes.
```
